boards/rp-pico/examples/pico_spi_sd_card.rs

```diff
@@ -251,34 +250,25 @@ fn main() -> ! {
     let spi = spi.init(
         &mut pac.RESETS,
         clocks.peripheral_clock.freq(),
-        16_000_000u32.Hz(),
+        400.kHz(), // card initialization happens at low baud rate
         &embedded_hal::spi::MODE_0,
     );
 
-    info!("Aquire SPI SD/MMC BlockDevice...");
-    let mut sdspi = SdMmcSpi::new(spi, spi_cs);
+    // We need a delay implementation that can be passed to SdCard and still be used
+    // for the blink signals.
+    let mut delay = &SharedDelay::new(cortex_m::delay::Delay::new(
+        core.SYST,
+        clocks.system_clock.freq().to_Hz(),
+    ));
 
-    blink_signals(&mut led_pin, &mut delay, &BLINK_OK_LONG);
-
-    // Next we need to aquire the block device and initialize the
-    // communication with the SD card.
-    let block = match sdspi.acquire() {
-        Ok(block) => block,
-        Err(e) => {
-            error!("Error retrieving card size: {}", defmt::Debug2Format(&e));
-            blink_signals_loop(&mut led_pin, &mut delay, &BLINK_ERR_2_SHORT);
-        }
-    };
-
-    blink_signals(&mut led_pin, &mut delay, &BLINK_OK_LONG);
-
-    info!("Init SD card controller...");
-    let mut cont = Controller::new(block, DummyTimesource::default());
+    info!("Initialize SPI SD/MMC data structures...");
+    let sdcard = SdCard::new(spi, spi_cs, delay);
+    let mut cont = Controller::new(sdcard, DummyTimesource::default());
```

Comment:
```suggestion
    let mut volume_mgr = VolumeManager::new(sdcard, DummyTimesource::default());
```

I renamed this type - the old name is for compatibility. Lots of rippling changes from this change will need doing too.

Reply:
Interesting, why didn't clippy warn me about that? Perhaps `#[deprecated]` doesn't work on `pub use` re-exports?